utils/flags/flags.go

```diff
@@ -219,6 +219,12 @@ func GetCliFlags() []cli.Flag {
 			Usage:   "The S3/minio bucket to use when using S3 proxy backend.",
 			EnvVars: []string{"BAZEL_REMOTE_S3_BUCKET"},
 		},
+		&cli.StringFlag{
+			Name:    "s3.bucket_lookup_type",
+			Value:   "auto",
+			Usage:   "The S3/minio bucket lookup type(auto|dns|path) to use when using S3 proxy backend.",
```

Comment:
nitpick: could you reword this to something like: `The S3/minio bucket lookup type. Allowed values: auto, dns, path.` and update the help text in README.md to match?